hackathon/templates/hackathon/includes/paginator.html

```diff
@@ -0,0 +1,18 @@
+        <ul class="pagination justify-content-center">
+            {% if page_obj.has_previous %}
```

Comment:
This is a really cool solution for pagination!

Reply:
Thanks!